### PR TITLE
TA-2977: Rename config export properties to match API

### PR DESCRIPTION
**Pull request by promeris**
#### Description

Changes are made to ensure that used properties match the provided API interface.

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [x] I have updated docs if needed

### CODE REVIEW

File: src/interfaces/package-export-transport.ts [hunk 1]
```diff
@@ -48,18 +48,16 @@ export interface PackageKeyAndVersionPair {
 export interface NodeExportTransport {
     key: string;
     parentNodeKey: string;
-    packageNodeKey: string;
```
Comment (ksalihu) [inline src/interfaces/package-export-transport.ts:51]: Why did we remove the package node key?

**Review by ksalihu — DISMISSED** [review]

File: src/interfaces/package-export-transport.ts [hunk 2]
```diff
@@ -48,18 +48,16 @@ export interface PackageKeyAndVersionPair {
 export interface NodeExportTransport {
```
Comment (ksalihu) [inline src/interfaces/package-export-transport.ts:48]: Should we also add the `PackageNodeExportTransport`?

File: src/interfaces/package-export-transport.ts [hunk 3]
```diff
@@ -48,18 +48,16 @@ export interface PackageKeyAndVersionPair {
...
     key: string;
     parentNodeKey: string;
-    packageNodeKey: string;
     name: string;
     type: string;
     exportSerializationType: string;
     serializedContent: string;
```
Comment (ksalihu) [inline src/interfaces/package-export-transport.ts:54]: Should we also rename `serializedContent` to `configuration`?

File: src/interfaces/package-export-transport.ts (hunk 1 above)
Comment (promeris) [reply src/interfaces/package-export-transport.ts:51]: The property is not used in CLI, and having it defined or not doesn't impact the parse/stringify steps during import.

File: src/interfaces/package-export-transport.ts (hunk 2 above)
Comment (promeris) [reply src/interfaces/package-export-transport.ts:48]: There is no benefit in doing this

**Review by Buqeta — DISMISSED** [review]

**Review by Buqeta — APPROVED** [review]

**Review by ksalihu — APPROVED** [review]

**Review by jetakasabaqi — APPROVED** [review]

**Pull request merged**